Docu for globalsearch (#1432)
SAP/luigi: docs/navigation-advanced.md

- **type**: boolean
- **description**: renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes you will get a drop-down menu in the horizontal navigation.

## Global search

> The global search element can be used to trigger a search. In your configuration you can implement an own search provider and some action events like onEnter, onEscape, onInput and onSearchResultItemSelected.

Reviewer: ```suggestion
The global search element can be used to trigger a search. In your configuration, you can implement a custom search provider and some action events like [onEnter](navigarion-parameters-reference.md#onEnter), [onEscape](navigarion-parameters-reference.md#onEscape), [onInput](navigarion-parameters-reference.md#onInput) and [onSearchResultItemSelected](navigarion-parameters-reference.md#onSearchResultItemSelected).
```

By default the search result will be rendered in a popup.

> Example:

Reviewer: It would be good to add a comprehensive example here and also maybe a screenshot above.

- **type**: boolean
- **description**: renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes you will get a drop-down menu in the horizontal navigation.

## Global search

> The global search element can be used to trigger a search. In your configuration, you can implement a custom search provider and some action events like [onEnter](navigarion-parameters-reference.md#onEnter), [onEscape](navigarion-parameters-reference.md#onEscape), [onInput](navigarion-parameters-reference.md#onInput) and [onSearchResultItemSelected](navigarion-parameters-reference.md#onSearchResultItemSelected).

Reviewer: ```suggestion
The global search element can be used to trigger a search. In your configuration, you can implement a custom search provider and some action events like [onEnter](navigation-parameters-reference.md#onEnter), [onEscape](navigation-parameters-reference.md#onEscape), [onInput](navigation-parameters-reference.md#onInput) and [onSearchResultItemSelected](navigation-parameters-reference.md#onSearchResultItemSelected).
```

Sorry for the typo 😆

> By default the search result will be rendered in a popup.

Reviewer: ```suggestion
By default, the search result will be rendered in a popup.
```

Example:

```javascript
onInput: () => {},
  searchProvider = {
    onInput: () => {
      let searchString = Luigi.globalSearch().getSearchString();

>       //Searchprovider do search

Reviewer: ```suggestion
      //searchProvider does a search
```

    },
    onEnter: () => {
      Luigi.globalSearch().showSearchResult(searchResults)
    },
    onEscape: () => {
      Luigi.globalSearch().closeSearchResult();
      Luigi.globalSearch().clearSearchField();
    },
    customSearchResultRenderer: (searchResults, slot, searchApiObj) => {

>       //If this function implemented the default search result popover will not be displayed.

Reviewer: ```suggestion
      //If this function is implemented the default search result popover will not be displayed.
```
